script/upgrades/eXOF/Config.sol

```diff
@@ -54,38 +56,39 @@ library eXOFConfig {
    */
   function CELOXOF_RateFeedConfig(
     Contracts.Cache storage contracts
-  ) internal returns (Config.RateFeed memory rateFeedConfig) {
+  ) internal view returns (Config.RateFeed memory rateFeedConfig) {
     rateFeedConfig.rateFeedID = contracts.deployed("StableTokenXOFProxy");
     rateFeedConfig.medianDeltaBreaker0 = Config.MedianDeltaBreaker({
       enabled: true,
       threshold: FixidityLib.newFixedFraction(3, 100), // 0.03
       cooldown: 30 minutes,
-      smoothingFactor: 0
+      smoothingFactor: 1e24 // 1
     });
-    rateFeedConfig.dependentRateFeeds = Arrays.addresses(contracts.dependency("EURXOFRateFeedAddr"));
+    rateFeedConfig.dependentRateFeeds = Arrays.addresses(Config.rateFeedID("EURXOF"));
```

Comment:
```suggestion
    rateFeedConfig.dependentRateFeeds = Arrays.addresses(Config.rateFeedID("EUROCXOF"));
```

Reply:
With the `EUROCXOF` dependency we should see the followoing behaviour:

1. `Scenario: |EUROC/EUR -1| > 0.005 & EUR/XOF  = 655.597`  => `CELO/eXOF` pool is suspended
2. `Scenario: |EUR/XOF -655.597| > 0.005 & EUROC/EUR =1`  => `CELO/eXOF` pool is suspended
3. `Scenario: EUROC/EUR * EUR/XOF = 655.597 & ( |EUROC/EUR - 1| > 0.005 & |EUR/XOF - 655.597| > 0.005 )` => `CELO/eXOF` is not suspended

Reply:
Just adding this comment to make sure we are on the same page with regard to setting & effects.
1. Is an unnecessary suspension but not a problem
2. Is as planned
3. Is the a weakness of not breaking `EUR/XOF`. But I think it is acceptable to start with

Reply:
As discussed offline, updated to have both `EUROC/XOF` and `EUR/XOF` as dependencies